sklearn/mixture/gmm.py

```diff
@@ -448,14 +455,20 @@ def fit(self, X):
             log_likelihood = []
             # reset self.converged_ to False
             self.converged_ = False
+
+            # this line should be removed when 'thresh' is deprecated
+            tol = self.tol if self.thresh is None \
+                else self.thresh / float(X.shape[0])
```

Comment:
From PEP8:

"""
The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation. 
"""